packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -774,16 +767,25 @@ export class SynapseTable extends React.Component<
                         name: column.name,
                       })}
                     >
-                      <Icon
-                        type={ICON_STATE[columnIndex]}
-                        cssClass={isSelectedIconClass}
-                      ></Icon>
+                      <IconSvg
+                        icon={ICON_STATE[columnIndex]}
+                        wrap={false}
+                        sx={{
+                          color: isSelected
+                            ? 'primary.contrastText'
+                            : 'primary.main',
+                          backgroundColor: isSelected ? 'primary.main' : 'none',
+
+                          '&:hover': {
+                            color: 'primary.contrastText',
+                            backgroundColor: 'primary.600',
+                          },
+                        }}
+                      />
                     </span>
                   )}
                   {isEntityIDColumn && (
-                    <span style={{ paddingBottom: '6px' }}>
-                      <EntityIDColumnCopyIcon />
-                    </span>
+                    <EntityIDColumnCopyIcon size={'small'} />
```

Comment:
Since this is now acts as  a basic MUI `IconButton`, we can just specify the `size` and remove the custom padding!